pallets/bridge/src/lib.rs

```diff
@@ -789,6 +850,29 @@ impl<T: Trait> Module<T> {
         Ok(())
     }
 
+    /// Applies a handler `f` to a vector of transactions `bridge_txs` and outputs a vector of
+    /// processing results.
+    fn apply_handler<F>(
+        f: F,
```

Comment:
```suggestion
        f: impl Fn(BridgeTx<T::AccountId, T::Balance>) -> DispatchResult,
```
(+ removing the `F` and `where`...)

(Not for this branch, but to be applied on `develop` at some point and as a reminder to self.)